### ISSUE TITLE
GFF files from annotation service are incorrectly formatted.

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1888]
Here is the entry for a gene that I annotated.  Note that it is missing the equal sign in the annotation in the gff output line.  I'm not sure about the escaped characters with the %2C but I am assuming those are also unnessary.

NODE_32_length_15125_cov_42.0081	rast2_export	CDS	12648	12995	.	+	0	ID=fig|590.16674.peg.5295;Name=Small multidrug resistance (SMR) efflux transporter > QacE delta 1%2C quaternary ammonium compounds

The annotation string should be:
"Small multidrug resistance (SMR) efflux transporter => QacE delta 1, quaternary ammonium compounds"

**Comment by aswarren** [comment]
Hey @jimdavis1 this in conformation to the GFF format.
Please see https://github.com/PATRIC3/patric3_website/issues/1443 and the associated link there.
Let us know if you want us to contravene the format specification (which will generally make it hard to parse into key value pairs).
